build.xml

```diff
@@ -263,6 +265,8 @@
         <string>--add-opens java.base/java.math=ALL-UNNAMED</string>
         <string>--add-opens java.base/java.lang.reflect=ALL-UNNAMED</string>
         <string>--add-opens java.base/java.net=ALL-UNNAMED</string>
+
+        <string>--add-opens java.rmi/sun.rmi.transport.tcp=ALL-UNNAMED</string>
```

Comment:
is this `--add-opens` scoped to the in-jvm dtests only?

Reply:
Neither is scoped specifically to in-jvm dtests, but they aren't in any production startup scripts either... it's only for IDE support and running with `ant test-run` which is documented as `Not for production use!` so I think it's appropriate to have them here.